### PR TITLE
Fix/ Search page: sort results by date modified

### PR DESCRIPTION
**Pull request by zbialecki**
Show most recently updated notes first in results.

Related to https://github.com/openreview/openreview/issues/149

### CODE REVIEW

File: pages/search.js [hunk 1]
```diff
@@ -134,10 +134,11 @@ const Search = ({ appContext }) => {
           content: query.content || 'all',
           group: query.group || 'all',
           source: Object.keys(sourceOptions).includes(query.source) ? query.source : 'all',
+          sort: 'tmdate:desc',
           limit: 1000,
         },
         null,
-        { accessToken, resultsKey: 'notes', sort: 'none' }
+        { accessToken, resultsKey: 'notes', sort: 'tmdate:desc' }
```
Comment (melisabok) [inline pages/search.js:141]: I wish we can sort them by pdate too, is there a way to decide this in the webfield component?

Comment (zbialecki) [reply pages/search.js:141]: This change is for the general /search page, not the venue homepage search.

**Review by melisabok — APPROVED** [review]

File: pages/search.js [hunk 2]
```diff
@@ -134,10 +134,11 @@ const Search = ({ appContext }) => {
           content: query.content || 'all',
           group: query.group || 'all',
           source: Object.keys(sourceOptions).includes(query.source) ? query.source : 'all',
+          sort: 'tmdate:desc',
```
Comment (xkopenreview) [inline pages/search.js:137]: i think only of sort here and sort in options is required?
it's taking options.sort or data1.sort

Comment (zbialecki) [reply pages/search.js:137]: Ah you are correct, I can remove it from the options object.

**Review by xkopenreview — APPROVED** [review]

**Pull request merged**